packages/core/src/utilities/display-visibility.scss

```diff
@@ -104,20 +104,13 @@ A breakpoint prefix is supported on all of the utility classes mentioned above,
 
 Markup:
 <% ['sm', 'md', 'lg', 'xl'].forEach(breakpoint => { %>
-  <div class="ds-u-<%= breakpoint %>-display--none ds-u-color--muted">
-    Hidden on <%= breakpoint %> screens and larger
-  </div>
-  <div class="ds-u-<%= breakpoint %>-display--block">
-    Displayed as block on <%= breakpoint %> screens and larger
-  </div>
-  <div class="ds-u-<%= breakpoint %>-display--flex">
-    Displayed as flex on <%= breakpoint %> screens and larger
-  </div>
-  <div class="ds-u-<%= breakpoint %>-visibility--hidden ds-u-color--muted">
-    Invisible on <%= breakpoint %> screens and larger
-  </div>
-  <div class="ds-u-<%= breakpoint %>-visibility--visible">
-    Visible on <%= breakpoint %> screens and larger
+  <div class="ds-u-margin--1">
```

Comment:
The original example was kind of hard to understand. Rather than showing the breakpoints with every visibility/display classes, I reduced it to just `hidden` and `visible`

Reply:
I'm thinking there might be some more work done here to help this be less confusing. I think we can file a ticket though for that future work.